src/main/java/cloud/fogbow/common/models/SystemUser.java

```diff
@@ -1,8 +1,16 @@
 package cloud.fogbow.common.models;
 
+import cloud.fogbow.common.constants.Messages;
+import cloud.fogbow.common.exceptions.UnexpectedException;
+import cloud.fogbow.common.util.GsonHolder;
+import cloud.fogbow.common.util.SerializedEntityHolder;
+
 import java.util.Objects;
 
 public class SystemUser extends User {
+
+    public static final int SERIALIZED_SYSTEM_USER_MAX_SIZE = 2048;
```

Comment:
cannot be private?

Reply:
Nope, ras is using it to set its the field size.